client/document/index.jsx

```diff
@@ -128,11 +128,12 @@ class Document extends React.Component {
 									[ 'is-section-' + sectionName ]: sectionName,
 									'is-jetpack-woocommerce-flow': isJetpackWooCommerceFlow,
 									'is-wccom-oauth-flow': isWCComConnect,
+									'is-jetpack-cloud-flow': config.isEnabled( 'jetpack-cloud' ),
 								} ) }
 							>
 								<div className="masterbar" />
 								<div className="layout__content">
-									{ 'jetpack-cloud' === sectionName ? (
+									{ config.isEnabled( 'jetpack-cloud' ) ? (
```

Comment:
Instead of `config.isEnabled( 'jetpack-cloud' )`
what if we had something that talks about the icon used for the loading screen. 

This way we could have that be jetpack specific for the cloud but in a different config it could be something else.